backend/api/tests/test_views.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import vcr
+from django.test import Client
+
+client = Client()
+
+@vcr.use_cassette()
+def test_api_root(client):
+    response = client.get('/api/v1/')
+    assert response.status_code == 200
+
+
+@vcr.use_cassette(record_mode='new_episodes')
+def test_businesses_endpoint(client):
```

Comment:
This tries to make a live call when I run it locally instead of using the cassette. Not sure why, but it's likely something to do with the way you recorded the cassette.

It's just an annoying and fairly common VCR quirk. As far as I can tell your implementation is spot on, but we should look into this together.